src/globals/test-extensions.php

```diff
@@ -33,14 +35,14 @@
 $no_strip = false;
 
 // compress with upx
-$upx = false;
+$upx = true;
 
 // prefer downloading pre-built packages to speed up the build process
-$prefer_pre_built = false;
+$prefer_pre_built = true;
```

Comment:
after some consideration, I believe this should also be false by default. I constantly switch between local musl, local glibc, alpine docker and gnu docker builds

Reply:
Same as above